src/core/server/http/router/router.ts

```diff
@@ -35,6 +36,11 @@ interface RouterRoute {
   handler: (req: Request, responseToolkit: ResponseToolkit) => Promise<ResponseObject | Boom<any>>;
 }
 
+export type RouteRegistrar = <P extends ObjectType, Q extends ObjectType, B extends ObjectType>(
+  route: RouteConfig<P, Q, B>,
+  handler: RequestHandler<P, Q, B>
+) => void;
+
```

Comment:
Outside of the scope of the initial ticket, but I added the `RouteRegistrar` type to avoid long type snippet duplication between `get/post/put/delete` methods. Commit is isolated, can remove if we don't want this